java-sdk-example/README.md

```diff
@@ -16,6 +16,8 @@ This example application uses Signadot's Java SDK to create a sandbox and test t
 The test creates a sandbox in the setup phase, runs tests against it and deletes the sandbox upon completion of the tests. Heres' the command to run it:
 
 ```
+export SIGNADOT_ORG=...
```

Comment:
Looks like it should be `SIGNADOT_ORG_NAME` based on the changes below.

Reply:
Done